doc/src/Transactions.dox

```diff
@@ -35,8 +41,8 @@
  *    manager.startTransaction(context)
  *    try:
  *        # Create the shot first, so we can then publish the clips to it
- *        shot_ref = manager.register([shot.name], [shot.attributes],
- *                                    [shot.spec], [entity_ref], context)[0]
+ *        shot_ref = manager.register([shot.name], [shot.spec],
+ *                                    [entity_ref], context)[0]
```

Comment:
[plan] What do we think about destructuring assignment?  I.e.
```suggestion
 *        [shot_ref] = manager.register([shot.name], [shot.spec],
 *                                    [entity_ref], context)
```

Reply:
Ahh yeah, that's much better isn't it, does that then also error if there is more than one with "too may values to unpack"

Reply:
Added a note to https://github.com/TheFoundryVisionmongers/OpenAssetIO/issues/287